trg-checks-dashboard/cmd/build.go

```diff
@@ -0,0 +1,182 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"log"
+	"math/rand"
+	"os"
+	"path"
+
+	cp "github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"trg-checks-dashboard/internal/templating"
+)
+
+const buildOutputDir = "build"
+const outputFileName = "index.html"
+
+// buildCmd represents the build command
+var buildCmd = &cobra.Command{
+	Use:   "build",
+	Short: "Create a statically compiled dashboard with release check status",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		ensureOutputDirExists()
+		copyAssets()
+
+		var outputBuffer bytes.Buffer
+		products, unhandledRepos := templating.CheckProducts()
+		// unhandledRepos := getDemoUnhandledRepos()
+
+		// templating.RenderHtmlTo(&outputBuffer, &templating.TemplateData{Config: getConfig(), CheckedProducts: getDemoChecks(), UnhandledRepos: unhandledRepos})
```

Comment:
Do we need these two if commented?

Reply:
yup indeed. just forgot to remove it. now gone!